.github/ISSUE_TEMPLATE/access-issues.yml

```diff
@@ -7,14 +7,29 @@ body:
 - type: markdown
   attributes:
     value: |
-      ⚠
-      This issue should be for problems accessing PyPI itself, including:
-      * pypi.org
-      * test.pypi.org
-      * files.pythonhosted.org
-  
-      This issue **should NOT** be for any non-PyPI properties (like
-      python.org, docs.python.org, etc.)
+      > [!CAUTION]
```

Comment:
*(This is a GFM syntax for one of those fancy admonitions that GH is able to render more prominently since https://github.com/orgs/community/discussions/16925)*

Reply:
Yep, when I first added these issue forms, I don't think that admonitions were supported. Hence the emoji.